spec/requests/posts_spec.rb

```diff
@@ -0,0 +1,7 @@
+require 'rails_helper'
+
+RSpec.describe 'Posts', type: :request do
+  describe 'GET /index' do
+    pending "add some examples (or delete) #{__FILE__}"
```

Comment:
- [ ] In this milestone students supposed to write test for controllers. But in your project I can see only pending message. Kindly write test for your both users and post: index and show. 

> HINT:
> Spec file include tests that ensures correct view template is rendered.
> Example : expect(response).to render_template(:show)
> 
> 
>